### DISCUSSION TITLE
Runtime warning when trying to plot data

### DISCUSSION THREAD

**Discussion by saharshBhargava** [geopandas/geopandas#3029] [Q&A]
Hello everyone. I'm using GeoPandas for the first time to plot some landslide data in the US i got from the USGS. I got the data from this website: https://www.sciencebase.gov/catalog/item/61f326dfd34e622189b93308

When I try to plot chunks of data (like 50 thousand values or so), it works and plots the points properly.
![image](https://github.com/geopandas/geopandas/assets/117414503/6a843692-f126-41ae-90db-8ad1128d5be2)

When I try to plot the whole thing (about 170000 data points), I get this error: /usr/local/lib/python3.10/dist-packages/matplotlib/axes/_base.py:2917: RuntimeWarning: overflow encountered in double_scalars x0, x1 = inverse_trans.transform([x0t - delta, x1t + delta]).

![image](https://github.com/geopandas/geopandas/assets/117414503/95f59a21-6bd2-4ff4-a7a7-8e4e85fed988)

So my question is what causes this error? Is it too many values? I checked the polygon data for empty values as well. Is it a device limitation? Does some number exceed the number of values numpy can store?

Thank you!

**Comment by martinfleis** [comment]
Check the coordinate values. Are all of them defined and within a reasonable range you'd expect?
